src/components/checkbox/checkbox.scss

```diff
@@ -12,6 +12,12 @@ $checkbox-top: 12px !default;
   }
 }
 
+// Needed for check to display correctly in IE11
+table md-checkbox {
```

Comment:
Since there isn't a md-table directive, I wasn't sure where to put it. Where would you like these styles to live? Would it be better to put it in md-checkbox?